README.md

```diff
@@ -16,3 +16,4 @@ Short Name    | Title                                                        | T
 [1/C4](1)     | Collective Code Construction Contract                        | Meta     | Raw        | Alberto Granzotto
 [2/COSS](2)   | Consensus-Oriented Specification System                      | Meta     | Raw        | Alberto Granzotto
 [3/UPSERT-VALIDATORS](3) | Dynamically add/remove validators at runtime (Insecure version)  | Standard | Raw | Vanshdeep Singh
+[5/STANDARIDIZE-DC](5) | Standard process to set up a local node for development & testing, using Docker Compose | Standard | Raw | Muawia Khan 
```

Comment:
The number should be `4`: `shortname: 4/STANDARIDIZE-DC`

Reply:
I kept it 5 because @ttmc had a comment that his PR https://github.com/bigchaindb/BEPs/pull/13, is numbered 4 and we might have a conflict. I can change, whatever you guys suggest. @ttmc @vrde